webapp/src/components/shareBoardComponent.tsx

```diff
@@ -76,8 +76,7 @@ const ShareBoardComponent = React.memo((props: Props): JSX.Element => {
     shareUrl.searchParams.set('r', readToken)
 
     if (match.params.workspaceId) {
-        const newPath = generatePath(':basename/workspace/:workspaceId/shared/:boardId/:viewId', {
-            basename: Utils.getFrontendBaseURL(),
+        const newPath = generatePath('plugins/focalboard/workspace/:workspaceId/shared/:boardId/:viewId', {
```

Comment:
I'm not 100% sure, but do we need a / at the beginning? Also, might be a separate issue, but will this work if the base Mattermost path is not on the root?

Reply:
Yes, I think we need the `/` at the beginning, but I don't know if the `shareurl.pathname = newPath` is already doing it.

Reply:
Nope...updated.